src/Microsoft.AspNet.Diagnostics/project.json

```diff
@@ -2,16 +2,16 @@
     "version": "1.0.0-*",
     "description": "ASP.NET 5 Middleware for error handling, error pages, and diagnostics information.",
     "dependencies": {
+        "Microsoft.AspNet.Diagnostics.Interfaces": "1.0.0-*",
```

Comment:
You don't seem convinced :smile: If the type is a regular type (*not* an ANI), why *wouldn't* a regular reference be needed? Where would the type come from, if not from that package?